### ISSUE TITLE
#1 setting CD 

### ISSUE THREAD

**Issue by kitsystemyou** [kitsystemyou/discordBot#1]
## Need setting Automatic push

refs https://nju33.com/glitch
* git push client
1. setting remote branch to @api.glitch/git/<application_name>
`git remote add glitch https://...@api.glitch.com/git/<application_name>`

* glitch
1. allow to push from local
`git config --add receive.denyCurrentBranch ignore`
1. setting auto refresh by post-receive huk
```
#! /bin/sh
cd /app;
git --git-dir=.git reset --hard
refresh
```

**Comment by kitsystemyou** [comment]
Then try to use Github actions
https://github.com/marketplace/actions/glitch-project-sync

**Comment by kitsystemyou** [comment]
Made workflow yaml.
Need to know access token of glitch. 
https://github.com/marketplace/actions/glitch-project-sync
Failed to get it 😞

**Comment by kitsystemyou** [comment]
~~Note:
Glitch's Access token has **no expire**. And Glitch has no refresh token.
https://www.glitchthegame.com/developer/docs/auth/~~

**Comment by kitsystemyou** [comment]
Got token. Ref : https://openbase.io/js/glitch-api/documentation
Execute in console  developer tool(F12).
`(JSON.parse(localStorage.getItem('cachedUser'))).persistentToken`

**Comment by kitsystemyou** [comment]
Complete auto push for glitch 🚀 
(Leave resolve needing "npm install twitter --save"

**Comment by kitsystemyou** [comment]
↑ Resolved by #21 from uehr
